pages/profiles.js

```diff
@@ -1,11 +1,14 @@
+import { useEffect, useState, useRef } from "react";
 import Link from "next/link";
 import {
   Box,
   ButtonBase,
-  Container,
   Grid,
   Typography,
+  TextField,
+  InputAdornment,
 } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
```

Comment:
```suggestion
import { Search } from 'mdi-material-design';
```

We use mdi-material-design library: https://materialdesignicons.com/ (it has a lot more icons)

Reply:
Ok, it's always good to know :)